tests/unit/.eslintrc.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
```

Comment:
wait a second, do we need another eslint here? I don't think so? It should just use the main one?

Reply:
this was definitely auto-generated, so idk.  in theory it could just use the main one but there are slightly different rules as I removed the errors from no-undef

Reply:
So apparently, under [this documentation](https://eslint.org/docs/user-guide/configuring) under the section `Configuration Cascading and Hierarchy`, it turns out that `ESLint` actually inherits previous `.eslintrc.js` files and attempts to merge the contents of the `.eslintrc.js` file in the current directory with any other encountered ones. 

This actually wasn't a necessary change then. @georgiashay Can this be restored to the original that was there before? Assuming I'm not remembering incorrectly, this should be the original contents of this `.eslintrc.js`:
```
module.exports = {
    env: {
      jest: true
  }
}

Reply:
This wasn't working for some reason, but I found the culprit:  For some reason, using extends: standard was messing with the jest linting plugin.  Do we need this, or could we just add the specific rules that we want?

Reply:
nevermind it just doesn't work